editor/components/inserter/menu.js

```diff
@@ -229,9 +228,9 @@ export class InserterMenu extends Component {
 		// If the Saved tab is selected and we have no results, display a friendly message
 		if ( 'saved' === tab && itemsForTab.length === 0 ) {
 			return (
-				<p className="editor-inserter__no-tab-content-message">
```

Comment:
Since this className `editor-inserter__no-tab-content-message` doesn't exist anymore, we should probably remove the corresponding style

Reply:
done :+1: